BunqSdk/Exception/ExceptionFactory.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+
+namespace Bunq.Sdk.Exception
+{
+    public class ExceptionFactory
+    {
+        private const int HTTP_RESPONSE_CODE_BAD_REQUEST = 400;
+        private const int HTTP_RESPONSE_CODE_UNAUTHORIZED = 401;
+        private const int HTTP_RESPONSE_CODE_FORBIDDEN = 403;
+        private const int HTTP_RESPONSE_CODE_NOT_FOUND = 404;
+        private const int HTTP_RESPONSE_CODE_METHOD_NOT_ALLOWED = 405;
+        private const int HTTP_RESPONSE_CODE_TOO_MANY_REQUESTS = 429;
+        private const int HTTP_RESPONSE_CODE_INTERNAL_SERVER_ERROR = 500;
+        
+        /// <summary>
+        /// Glue to concatenate the error messages.
+        /// </summary>
+        private const string GLUE_ERROR_MESSAGES = "\n";
+
+
+
+        /// <param name="responseCode"></param>
+        /// <param name="messages"></param>
+        /// <returns>The exception that belongs to this status code.</returns>
+        public static ApiException CreateExceptionForResponse(int responseCode, IList<string> messages)
```

Comment:
We do not need the param tags here! Actually, a normal summary would be just good here :)

Reply:
Isn't `/// <returns>The exception that belongs to this status code.</returns>`  enough ? 🤔

Reply:
Enough - yes, but we do not use `returns`. We just put things in summary at this point :). Please take a look at the rest of C# SDK for reference!